tools/accuracy_checker/accuracy_checker/metrics/score_class_comparison.py

```diff
@@ -0,0 +1,98 @@
+"""
+Copyright (c) 2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .metric import FullDatasetEvaluationMetric
+from ..config import NumberField
+from ..representation import QualityAssessmentAnnotation, QualityAssessmentPrediction
+
+class ScoreClassComparisonMetric(FullDatasetEvaluationMetric):
+    __provider__ = 'score_class_comparison'
+    annotation_types = (QualityAssessmentAnnotation, )
+    prediction_types = (QualityAssessmentPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'num_high_quality': NumberField(
+                value_type=int,
+                min_value=1,
+                optional=False,
+                description="The number of high-quality class in total"
+            ),
+            'num_low_quality': NumberField(
+                value_type=int,
+                min_value=1,
+                optional=False,
+                description="The number of low-quality class in total"
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.high_class = self.get_value_from_config('num_high_quality')
+        self.low_class = self.get_value_from_config('num_low_quality')
+
+    def evaluate(self, annotations, predictions):
+        if len(predictions) < self.high_class or len(predictions) < self.low_class:
+            return 0.0
+
+        # Sort by quality score
+        quality_scores = np.array([a.quality_score for a in annotations])
+        quality_scores[::-1].sort()
```

Comment:
not sure that it works...
if I right understand, do you need to sort array in descent order?
now you revert array and then sort, so order will be the same like without [::-1]

Reply:
It works. `quality_scores[::-1]` changes its view on quality_scores array rather than reversing its order. `.sort()` method sorts the array in-place with the changed view.

This short python commands will help you to understand it works.

```python
>>> import numpy as np
>>> a = np.array([2,1,3,5,4])
>>> a.sort()
>>> a
array([1, 2, 3, 4, 5])
>>> b = np.array([2,1,3,5,4])
>>> b[::-1].sort()
>>> b
array([5, 4, 3, 2, 1])
```

Reply:
then ok, thanks!